### PR TITLE
KOGITO-1899: BDD tests: Test travel-agency scenario in native mode

### PR DESCRIPTION
**Pull request by Sgitario**
JIRA Ticket: https://issues.redhat.com/browse/KOGITO-1899
Description: Configure Travel Agency tests with native enabled

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: test/features/deploy_travel_agency.feature [hunk 1]
```diff
@@ -52,7 +54,32 @@ Feature: Deploy Travel agency service and verify its functionality
...
+	Examples:
+      | native   | minutes | build-request-cpu | 
+      | disabled | 10      | 1                 | 
+
+    # Disabled as long as https://issues.redhat.com/browse/KOGITO-1893  is not merged or https://issues.redhat.com/browse/KOGITO-1917 is not fixed
+    @disabled
+    @native
+    Examples:
+      | native   | minutes | build-request-cpu | 
+      | enabled  | 30      | 4                 |
```
Comment (Sgitario) [inline test/features/deploy_travel_agency.feature:66]: With CPU 4, the build takes more than 20 min. That's why I set 30 min here.
Without CPU, the build takes more than 30 min).

**Comment by Sgitario** [comment]
I fixed all the indentation issues.
I've run the tests in quarkus native with the workaround and they worked fine.

File: test/features/deploy_travel_agency.feature [hunk 2]
```diff
@@ -52,7 +54,32 @@ Feature: Deploy Travel agency service and verify its functionality
 
     Then Service "travels" with process name "travels" is available
 
-  Scenario: Travel application with required Visa
+	Examples:
```
Comment (radtriste) [inline test/features/deploy_travel_agency.feature:57]: indentation :D

File: test/features/deploy_travel_agency.feature [hunk 3]
```diff
@@ -116,3 +143,14 @@ Feature: Deploy Travel agency service and verify its functionality
 	  """
 
     Then Service "travels" with process name "travels" is available
+
+	Examples:
```
Comment (radtriste) [inline test/features/deploy_travel_agency.feature:147]: indentation

File: test/features/deploy_travel_agency.feature (hunk 2 above)
Comment (Sgitario) [reply test/features/deploy_travel_agency.feature]: Fixed. I've already configured my VS to avoid this issue in the future.

**Comment by Sgitario** [comment]
@MarianMacik can you have a look to this PR? Thanks

**Comment by radtriste** [comment]
waiting for https://github.com/kiegroup/kogito-examples/pull/257 to be merged

**Pull request merged**